src/main/java/seedu/address/model/item/field/Time.java

```diff
@@ -15,6 +15,7 @@
 
     public static final String MESSAGE_CONSTRAINTS = "Time should be in the format MM-yyyy";
     public static final String VALIDATION_REGEX = "^(1[0-2]|0[1-9])-[0-9]{4}$";
+    public static final String MESSAGE_FROM_TO_MISORDER = "\"from\" field cannot be after \"to\" field. ";
```

Comment:
Same comment that Duong made on mine lol, "\'from\' field cannot be later \'to\' field.". It's cuz the word "after" can be confusing